### PR TITLE
Adding noParse on module

### PR DESCRIPTION
**Pull request by psachs21**
Related to #26 

Will need to update @types once changed.

### CODE REVIEW

**Comment by psachs21** [comment]
Little bit confused by errors in codacy, given the changes are consistent with existing code base.

File: src/Module.js [hunk 1]
```diff
@@ -33,6 +36,9 @@ module.exports = class extends ChainedMap {
               .keys(value)
               .forEach(name => this.rule(name).merge(value[name]));
           }
+          case 'noParse': {
```
Comment (eliperelman) [inline src/Module.js]: Insert 1 blank line before this `case`.

**Comment by eliperelman** [comment]
@psachs21 don't worry about the codacy errors, I will get this merged. 😃

**Comment by psachs21** [comment]
@eliperelman thanks!

**Comment by eliperelman** [comment]
Released in v3.3.0! Thank you!

**Pull request merged**